src/api/rif-notifier-service/subscriptionPlans.ts

```diff
@@ -25,4 +25,9 @@ export default class SubscriptionPlans
     }
     return isValidResponse
   }
+
+  _create = async <SubscriptionPlanDTO>(data: SubscriptionPlanDTO): Promise<boolean> => {
+    await this.service.create(data)
+    return true
+  }
```

Comment:
Why returning true, or in fact why type it so arbitrarily? Maybe just
```suggestion
  _create = <T>(data: T): Promise<unknown> => this.service.create(data)
```
would do? Or, since this endpoint never uses it even:
```suggestion
  _create!: <T, S>(data: T, params?: S | undefined) => Promise<any>
```
would suffice. Or, cleaner still:
```suggestion
  _create = (): Promise<void> => Promise.resolve(logNotImplemented('Subscription Plans')())
```